include/ofi_mr.h

```diff
@@ -241,6 +242,7 @@ struct ofi_mr_cache_params {
 	size_t				max_cnt;
 	size_t				max_size;
 	char *				monitor;
+	int				cuda_monitor_enabled;
```

Comment:
There should be a more general way of marking each monitor as enabled/disabled.  Something that applies to memhooks, uffd, etc.